### PR TITLE
Issue #345 server returns last bc211 update date 

### PR DESCRIPTION
**Pull request by tomy-pg**

### CODE REVIEW

File: BC211_LAST_UPDATE.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+20-03-2019
```
Comment (rasmus-storjohann-PG) [inline BC211_LAST_UPDATE.txt]: Remove

File: bc211/version.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# The version is defined in /pathways-backend/bc211/__init__.py
```
Comment (rasmus-storjohann-PG) [inline bc211/version.py]: Regarding the file name, if you look at existing filenames, you'll find that they are named after the kinds of classes they contain. So this file should be called views.py.

Comment (tomy-pg) [reply bc211/version.py]: Fixed. I was following the existing file names in /main when I named it version.py.

Comment (rasmus-storjohann-PG) [reply bc211/version.py]: Yeah, that file name is not very consistent. /main is a bit of a grab bag folder, unfortunately.

**Pull request merged**